README.md

```diff
@@ -41,7 +41,10 @@ This is the dependency you need:
 </dependency>
 ```
  
-Then, you implement `com.yegor256.rpm.Storage` interface.
+Then, you implement `com.artipie.asto.Storage` interface or use one the existing implementations,
```

Comment:
@olenagerasimova I'd suggest to switch existing options and `you implement`. Obviously, own implementation of `Storage` is not a likely scenario